packages/smart/funcs.go

```diff
@@ -709,7 +709,10 @@ func ColumnCondition(sc *SmartContract, tableName, name, coltype, permissions, i
 	if isExist {
 		return sc.AccessTable(tblName, `update`)
 	}
-	count, _ := model.GetColumnCount(tblName)
+	count, err := model.GetColumnCount(tblName)
+	if err != nil {
+		return err
```

Comment:
You must log it here, because there is a file and line fields in logs, which should point to this location